components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -183,26 +185,36 @@ subroutine lin_strat_chem_inti(phys_state)
 
     ! define additional output
 
-    call addfld( 'LINOZ_DO3'    , (/ 'lev' /), 'A', '/s'     , 'ozone vmr tendency by linearized ozone chemistry'   )
-    call addfld( 'LINOZ_DO3_PSC', (/ 'lev' /), 'A', '/s'     , 'ozone vmr loss by PSCs using Carille et al. (1990)' )
+    call addfld( 'LINOZ_DO3LNZ'    , (/ 'lev' /), 'A', '/s'     , 'ozone vmr tendency by linearized ozone chemistry'   )
+    call addfld( 'LINOZ_DO3LNZ_PSC', (/ 'lev' /), 'A', '/s'     , 'ozone vmr loss by PSCs using Carille et al. (1990)' )
+    call addfld( 'LINOZ_2DDO3LNZ'    , horiz_only, 'A', 'DU/s'     , 'ozone 2D DU tendency by linearized ozone chemistry'   )
+    call addfld( 'LINOZ_2DDO3LNZ_PSC', horiz_only, 'A', 'DU/s'     , 'ozone 2D DU loss by PSCs using Carille et al. (1990)' )
```

Comment:
Are these output for "O3LNZ" or "O3"? I suggest changing them to "O3LNZ 2D DU ..."

Reply:
Modified.